receiver/mongodbreceiver/metadata.yaml

```
@@ -1 +1,143 @@
 name: mongodbreceiver
+
+attributes:
+  database:
+    description: The name of a database.
+  memory_type:
+    value: type
+    description: The type of memory used.
+    enum:
+      - resident
+      - virtual
+  operation:
+    description: The mongoDB operation being counted.
```

```suggestion
    description: The MongoDB operation being counted.
```
Can I also ask that you review the readme for inconsistent usage of `MongoDB`? I see at least two lines needing correction at the top of the doc.

---

done 👍